### ISSUE TITLE
How to organize metadata exchange between entities

### ISSUE THREAD

**Issue by surferau** [sschmid/Entitas-CSharp#673]
Hello, coming from MVP i have a question how to organize the following situation:

say i have a tile with position x,y; in this tile there is a car with manual shift, painted red and price $100. There is also driver, so 2 entities like this:

class carClass
{
shift, color
}

var newCar = new carClass(manual, red);

var carEntity = createentity;
carEntity.addPosition(x,y); //common data applied to other entities as well
carEntity.addPrice(100); //common data applied to other entities as well
carEntity.addCar(newCar );

var driverEntity = createenity;
driverEntity .isDriver = true;
and here is the question, it can be like this
driverEntity .addCarEntity(carEntity); // we have all metadata like position and other stuff. when do 
//driverEntity.carEntity.value.price.value
OR
driver.addCar(newCar ); //we dont know position and price, so we have to make a lookup call something like
//context.getentitieswithcar(newcar);

TLDR
Is it a good idea to keep referencing other entities in entity components or i should avoid this and extract metadata with lookup functions or there is another approach?

**Comment by surferau** [comment]
Thank you
